src/utils/vm_mount.cpp

```diff
@@ -58,6 +58,51 @@ mp::VMMount::VMMount(const std::string& sourcePath,
       uid_mappings(std::move(uidMappings)),
       mount_type(mountType)
 {
+    fmt::memory_buffer errors;
+
+    auto print_mappings = [](auto& dup_id_map, const auto& dup_rev_id_map) {
```

Comment:
Why a lambda? You're not capturing anything, so no closure. I like that you have a function, but I would prefer to see it in the unnamed namespace for this file. The reader of this constructor (or any function) will most often want to see _what_ it is doing, not _how_ it is doing it. If they want to see the _what_ part, they can always zone in on what interests them.